src/ConductorSharp.Client/Model/Common/WorkflowDefinition.cs

```diff
@@ -14,8 +14,21 @@ public class SubWorkflowParam
             [JsonProperty("name")]
             public string Name { get; set; }
 
+            [JsonIgnore]
+            public int Version
+            {
+                get
+                {
+                    if (int.TryParse(VersionAsString, out int version))
+                        return version;
+
+                    return 0;
+                }
+                set { VersionAsString = value.ToString(); }
+            }
+
             [JsonProperty("version")]
-            public int Version { get; set; }
+            public string VersionAsString { get; set; }
```

Comment:
Why is this changed to string?

Reply:
Dynamic tasks can have their version passed in as a parameter bound from a task or workflow input so we must support both options.